contents/docs/toolbar/index.mdx

```diff
@@ -39,4 +39,6 @@ if (toolbarJSON) {
 }
 ```
 
-We are also working on other solutions. Go to [the related issue](https://github.com/PostHog/posthog/issues/9100) to see progress updates (and encourage us to get it done faster!).
\ No newline at end of file
+We are also working on other solutions. Go to [the related issue](https://github.com/PostHog/posthog/issues/9100) to see progress updates (and encourage us to get it done faster!).
+
+> **Nuxt issues:** Nuxt overrides the URL of our static asset URL from `https://app-static-prod.posthog.com` to `https:/app-static-prod.posthog.com` removing the second `/`. This causes the toolbar to not load. If you have a solution for this, let us know via [the support modal](https://app.posthog.com/home#supportModal=feedback%3A).
```

Comment:
```suggestion
> **Warning:** The PostHog toolbar currently doesn't work for Nuxt users because Nuxt overrides the URL of our static asset URL from `https://app-static-prod.posthog.com` to `https:/app-static-prod.posthog.com`, removing the second `/`. If you have a solution, [please let us know](https://app.posthog.com/home#supportModal=feedback%3A).
```